XSUP-35612/ Fixing mirroring issues (#33806)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/1_13_6.md

#### Mappers

##### CrowdStrike Falcon Mapper

> Updated the **name** incident field in the Common Mapping, to match the rest of the incident types.

Reviewer: ```suggestion
Updated the **name** incident field in the Common Mapping, to be aligned with other incident types.
```

#### Integrations

##### CrowdStrike Falcon

> Added more fields to the mirroring in process.

Reviewer: Please specify which fields
